synce_ql_manager.c

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file synce_ql_manager.c
+ * @brief Interface for managing QL for external clock sources
+ * @note SPDX-FileCopyrightText: Copyright 2022 Intel Corporation
+ * @note SPDX-License-Identifier: GPL-2.0+
+ */
+#include <unistd.h>
+#include <pthread.h>
+#include <stdlib.h>
+#include <net/if.h>
+#include "print.h"
+#include "config.h"
+#include "synce_dev.h"
+#include "synce_clock.h"
+#include "synce_ql_manager.h"
+
+#define SOCKET_PATH "/tmp/synce4l_socket"
```

Comment:
I dont think it is correct. the synce4l manages all instances. BTW, this is same concept of ptp4l

Reply:
I know it can manage multiple devices and was designed this way, but the capabilities is one thing and users and their requirements is different, with this patch we disallow running multiple instances, which can be considered regression/bug for some users, the config items do have a default value for the parameters, so we could use a default, but still allow the change without recompiling the code.

TLV based api would get some parsing out of our scope, but I don't have any strong opinion on that, If you prefer strings go for it ;) , but please make sure those are parsed for safety/validated in one place in a common way and then already parsed/validated values are actual arguments for invoking config change, what i mean let's not parse strings out of ql manager,  already parsed strings/values shall be args for getting pointers to the objects that are to be changed and once all is good, the change shall be invoked.

Reply:
added to configuration file